torch/_refs/__init__.py

```diff
@@ -3987,7 +3986,6 @@ def index_select(x: TensorLike, dim: int, index: TensorLike):
     return x[idx]
 
 
-@register_decomposition(aten.squeeze)
```

Comment:
We can still register these for `sum,dim_IntList` and `squeeze.dims` which should fix the decomp test.

Reply:
ah, good point. Let me update.